src/pathfinding_service/api.py

```diff
@@ -297,7 +302,9 @@ def post(
         token_network = self._validate_token_network_argument(token_network_address)
         feedback_request = self._parse_post(FeedbackRequest)
         feedback_token = self.pathfinding_service.database.get_feedback_token(
```

Comment:
Instead of doing
* select
* check if feedback is present
* update

you could do all in one step:
```
UPDATE feedback
SET successful = ?, feedback_time = ? 
WHERE token_id = ? AND route = ? AND successful IS NULL
```
The results `rowcount` attribute will tell you how many rows have been changed.

Reply:
That's a nice improvement!